script/build_example_apps

```diff
@@ -245,9 +243,6 @@ main()
 
     options+=("-DCMAKE_TOOLCHAIN_FILE=src/${platform}/arm-none-eabi.cmake")
 
-    # Find slc-cli installation
-    slc_init
```

Comment:
Did this command have a purpose previously and is no longer used, or is this change cleaning up dead code?

Reply:
This is cleaning up dead code. I moved the SLC calls to `script/generate` a while ago and forgot to remove this